pango/src/matrix.rs

```diff
@@ -0,0 +1,34 @@
+// Take a look at the license at the top of the repository in the LICENSE file.
+
+use crate::{Matrix, Rectangle};
+use glib::translate::*;
+
+impl Matrix {
+    pub fn new(xx: f64, xy: f64, yx: f64, yy: f64, x0: f64, y0: f64) -> Self {
+        Self(ffi::PangoMatrix {
+            xx,
+            xy,
+            yx,
+            yy,
+            x0,
+            y0,
+        })
+    }
+
+    #[doc(alias = "pango_matrix_transform_pixel_rectangle")]
+    pub fn transform_pixel_rectangle(&self, mut rect: Option<&mut Rectangle>) {
+        unsafe {
+            ffi::pango_matrix_transform_pixel_rectangle(
+                self.to_glib_none().0,
+                rect.to_glib_none_mut().0,
+            )
+        }
+    }
+
+    #[doc(alias = "pango_matrix_transform_rectangle")]
+    pub fn transform_rectangle(&self, mut rect: Option<&mut Rectangle>) {
```

Comment:
Put this and the other new function into `manual = true` in the `Gir.toml` so that the commented out autogenerated code disappears and docs are generated.